src/audiomixerboard.cpp

```diff
@@ -1174,30 +1174,27 @@ void CAudioMixerBoard::ApplyNewConClientList ( CVector<CChannelInfo>& vecChanInf
                         }
                     }
 
-                    // restore gain (if new name is different from the current one)
-                    if ( vecpChanFader[i]->GetReceivedName().compare ( vecChanInfo[j].strName ) )
```

Comment:
The diff looks larger than it is. Essentially, this `if` is replaced by an `if ( true ) `. `git show -w bd081647b2152f9a430c4c59bb0b98a035d2cade` will be more readable than Github. :)

Reply:
Why doesn't Github have an "ignore whitespace".....?!

Reply:
Just learned that Github actually supports this! :)

https://github.blog/2018-05-01-ignore-white-space-in-code-review/

https://github.com/jamulussoftware/jamulus/pull/1009/files?w=1